net/pfSense-pkg-haproxy-devel/files/usr/local/www/haproxy/haproxy_global.php

```diff
@@ -29,9 +29,10 @@
 require_once("haproxy/haproxy_htmllist.inc");
 require_once("haproxy/pkg_haproxy_tabs.inc");
 
-$simplefields = array('nbthread', 'hard_stop_after', 'localstats_refreshtime', 'localstats_sticktable_refreshtime', 'log-send-hostname', 'ssldefaultdhparam',
-  'email_level', 'email_myhostname', 'email_from', 'email_to',
-  'resolver_retries', 'resolver_timeoutretry', 'resolver_holdvalid');
+$simplefields = array('nbthread', 'hard_stop_after', 'localstats_refreshtime', 'localstats_sticktable_refreshtime',
+	'log-send-hostname', 'sslcompatibilitymode', 'ssldefaultdhparam', 'email_level', 'email_myhostname',
+	'email_from', 'email_to', 'resolver_retries', 'resolver_timeoutretry', 'resolver_holdvalid');
+$sslcompatibilitymodes = array('auto' => 'Auto', 'old' => 'Old', 'intermediate' => 'Intermediate', 'modern' => 'Modern');
```

Comment:
The order of the non-auto options should be reversed, so: Auto, Modern, Intermediate, Old. Put old last since it should (hopefully) be the least used option.

Reply:
Ok, fixed!